bitnami/mongodb/Chart.yaml

```
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mongodb
   - https://mongodb.org
-version: 13.0.0
+version: 13.0.1
```

We are adding a new feature for replicaset architecture, according to [semver](https://semver.org/) we should bump a new minor version:

```suggestion
version: 13.1.0
```